app/namespace.go

```diff
@@ -269,6 +270,32 @@ func (c *NamespaceClient) createNamespace(n *namespace.Namespace, p []*auth.User
 	return PrintProto(res)
 }
 
+func (c *NamespaceClient) addReplica(ctx *cli.Context) error {
+	var resourceVersion string
+	if v := ctx.String(ResourceVersionFlagName); v != "" {
+		resourceVersion = v
+	}
+
+	region := ctx.String(namespaceRegionFlagName)
+	if len(region) == 0 {
+		return fmt.Errorf("namespace region is required")
+	}
+
+	res, err := c.client.GlobalizeNamespace(c.ctx, &namespaceservice.GlobalizeNamespaceRequest{
+		RequestId: ctx.String(RequestIDFlagName),
+		Namespace: ctx.String(NamespaceFlagName),
+		TargetRegion: &common.RegionID{
+			CloudProvider: "aws",
```

Comment:
maybe add TODO: to also require cloud when we get to it

Reply:
I will add a cloud provider input flag.